force-app/main/default/classes/SOQL.cls

```diff
@@ -66,6 +88,16 @@ public with sharing class SOQL {
     //[public | private ] [static] data_type method_name (input parameters) {
     //  The body of the method
     //}
+    public static List<Opportunity> getClosedWonOpportunityCountByAccount (Id accountId){
+        List<Opportunity> opp = [SELECT Id, StageName, AccountId, Name, Description
+        FROM Opportunity
+        WHERE StageName = 'Closed Won' AND AccountId =: accountId
+        Limit 1];
+
+        return opp;
```

Comment:
This is the first place where you'll want to start thinking about query formatting. As your queries get bigger and more complex, you will want to start formatting them vertically so they are easier to read.
```suggestion
        List<Opportunity> opps = [
                SELECT 
                        Id,
                        Name,
                        StageName,
                        AccountId,
                        CloseDate,
                        Amount 
                FROM Opportunity 
                WHERE AccountId = :accId 
                AND StageName = 'Closed Won'
        ];
```